.github/workflows/test.yml

```diff
@@ -185,27 +174,4 @@ jobs:
 #        uses: actions/upload-artifact@v3
 #        with:
 #          name: redshift-test-results
-#          path: build/spock-reports
-
-  dependabot:
```

Comment:
This dependabot merge part... was it moved elsewhere?

Reply:
added back here https://github.com/liquibase/liquibase-cassandra/pull/218/commits/d5004627eb090b34cb6bce99fe4dfd20837cce17

Reply:
one more thing :) It should need "unit-test" , not build - so it will only merge if tests are passing .

Reply:
done @filipelautert